### PR TITLE
Add option to reveal previous card's info

### LINKED ISSUE #1355: Add show previous card info button to anki reviewer
Following my old issue, https://github.com/ankitects/anki/issues/1347 (https://github.com/evandroforks/anki/commit/a6474a0d171c8ad4a44333ae52424d02bda1dd64), another button I added was the previous card button. I added this because I mostly answer the card right away and only later do I worry about seen its history. Without this button, I had to hit `Ctrl+Z` to undo my review and get the old card back so I could see its status (and answer it back 😢).

```patch
Subject: [PATCH] Created the previous card info button (for the last card)

---
 qt/aqt/reviewer.py | 13 ++++++++++++-
 1 file changed, 12 insertions(+), 1 deletion(-)

diff --git a/qt/aqt/reviewer.py b/qt/aqt/reviewer.py
index f0749974..77f38e75 100644
--- a/qt/aqt/reviewer.py
+++ b/qt/aqt/reviewer.py
@@ -47,6 +47,7 @@ class Reviewer:
         self.mw = mw
         self.web = mw.web
         self.card: Optional[Card] = None
+        self.lastcard: Optional[Card] = None
         self.cardQueue: List[Card] = []
         self.hadCardQueue = False
         self._answeredIds: List[int] = []
@@ -108,6 +109,7 @@ class Reviewer:
                 self.mw.col.reset()
                 self.hadCardQueue = False
             c = self.mw.col.sched.getCard()
+        self.lastcard = self.card
         self.card = c
         if not c:
             self.mw.moveToState("overview")
@@ -121,6 +123,13 @@ class Reviewer:
     ##########################################################################
 
     def cardInfo(self) -> None:
+        self._cardInfo(self.card)
+
+    def lastCardInfo(self) -> None:
+        if self.lastcard:
+            self._cardInfo(self.lastcard)
+
+    def _cardInfo(self, card) -> None:
         from anki.stats import CardStats
         from aqt.qt import QDialog
         from PyQt5 import QtCore, QtGui, QtWidgets
@@ -128,7 +137,7 @@ class Reviewer:
         from aqt import QDialogButtonBox, qconnect, Qt
         from aqt.utils import saveGeom, restoreGeom
 
-        cs = CardStats(self.mw.col, self.mw.reviewer.card)
+        cs = CardStats(self.mw.col, card)
         info = cs.report(include_revlog=True)
         reps = ""
 
@@ -335,6 +344,7 @@ class Reviewer:
             ("r", self.replayAudio),
             (Qt.Key_F5, self.replayAudio),
             ("Ctrl+i", self.cardInfo),
+            ("Ctrl+o", self.lastCardInfo),
             ("Ctrl+1", lambda: self.setFlag(1)),
             ("Ctrl+2", lambda: self.setFlag(2)),
             ("Ctrl+3", lambda: self.setFlag(3)),
@@ -785,6 +795,7 @@ time = %(time)d;
             # [_("Options"), "O", self.onOptions],
             None,
             [_("Card Info"), "Ctrl+I", self.cardInfo],
+            [_("Last Card Info"), "Ctrl+P", self.lastCardInfo],
             [_("Replay Audio"), "R", self.replayAudio],
             [_("Pause Audio"), "5", self.on_pause_audio],
             [_("Audio -5s"), "6", self.on_seek_backward],
-- 
2.28.0.windows.1
```

### PR DESCRIPTION
**Pull request by evandrocoan**

### CODE REVIEW

File: qt/aqt/reviewer.py [hunk 1]
```diff
@@ -456,6 +457,7 @@ def _shortcutKeys(
             ("Shift+v", self.onRecordVoice),
             ("o", self.onOptions),
             ("i", self.on_card_info),
+            ("Ctrl+i", self.on_previous_card_info),
```
Comment (dae) [inline qt/aqt/reviewer.py:460]: Alt feels like a more natural choice for an alternate/modified action that ctrl/cmd to me - was there a technical reason/conflict that prevented you from using it?

**Comment by dae** [comment]
Thanks Evandro, looks fine apart from the shortcut

File: qt/aqt/reviewer.py (hunk 1 above)
Comment (evandrocoan) [reply qt/aqt/reviewer.py]: Alt is just hard for my hands to press, then, I press first using Ctrl and if not available go to Alt. But on this case, other than my hand issues, I think Alt could be better to easily confuse a user that things Ctrl+i is for the current card.

**Comment by dae** [comment]
Thanks!

**Pull request merged**